client/components/deposits-overview/deposit-notices.tsx

```diff
@@ -222,30 +223,38 @@ export const DepositFailureNotice: React.FC< {
 	 * The link to update the account details.
 	 */
 	updateAccountLink: string;
-} > = ( { updateAccountLink } ) => (
-	<InlineNotice
-		status="warning"
-		icon
-		className="deposit-failure-notice"
-		isDismissible={ false }
-	>
-		{ interpolateComponents( {
-			mixedString: __(
-				'Deposits are currently paused because a recent deposit failed. Please {{updateLink}}update your bank account details{{/updateLink}}.',
-				'woocommerce-payments'
-			),
-			components: {
-				updateLink: (
-					<ExternalLink
-						onClick={ () =>
-							recordEvent( 'wcpay_account_details_link_clicked', {
-								source: 'deposit-notices',
-							} )
-						}
-						href={ updateAccountLink }
-					/>
+} > = ( { updateAccountLink } ) => {
+	const accountLinkWithSource = addQueryArgs( updateAccountLink, {
+		source: 'deposit-notices',
```

Comment:
Maybe use a source like `deposits-overview__deposit-failure-notice`

Reply:
Addressed in https://github.com/Automattic/woocommerce-payments/pull/8668/commits/6805615ab089eb023e57737d80ae7d96248ed2c4